include/library/spdm_common_lib.h

```diff
@@ -232,16 +232,41 @@ void libspdm_set_last_spdm_error_struct(void *spdm_context,
                                         libspdm_error_struct_t *last_spdm_error);
 
 /**
- * Initialize an SPDM context.
+ * Initialize an SPDM context, as well as all secured message contexts,
+ * in the specified locations.
  *
- * The size in bytes of the spdm_context can be returned by libspdm_get_context_size.
+ * The size in bytes of the spdm_context can be returned by
+ * libspdm_get_context_size_without_secured_context.
  *
- * @param  spdm_context  A pointer to the SPDM context.
+ * The size in bytes of a single secured message context can be returned by
+ * libspdm_secured_message_get_context_size.
+ *
+ * @param  spdm_context          A pointer to the SPDM context.
+ * @param  secured_contexts      An array of pointers, with each entry containing
+ *                               the location of a secured message context.
+ * @param  num_secured_contexts  Number of secured message contexts to initialize.
+ *                               Currently, only LIBSPDM_MAX_SESSION_COUNT is supported.
```

Comment:
Also state that `num_secured_contexts` may be less than `LIBSPDM_MAX_SESSION_COUNT` in a future release.

Reply:
Added